lib/traject/annotation_config.rb

```diff
@@ -0,0 +1,76 @@
+require 'traject_plus/macros'
+
+# rubocop:disable Style/MixinUsage
+extend TrajectPlus::Macros
+# rubocop:enable Style/MixinUsage
+
+settings do
+  provide 'processing_thread_pool', 1
+end
+
+each_record do |record, context|
+  context.clipboard[:annotation] = ::JSON.parse(record.data)
+  context.clipboard[:canvas] = ::JSON.parse(Faraday.get(record.canvas).body)
+  context.clipboard[:manifest] = ::JSON.parse(Faraday.get(context.clipboard[:annotation]['on'].first['within']['@id']).body)
+end
+
+to_field 'id', (accumulate { |resource, *_| resource.uuid })
```

Comment:
Ugh. Are all these annotations under our control, and is there an opportunity to remediate them? Otherwise, I'll just has the id and call it a day.

Reply:
Yes at the moment all inside of colligo/annotot stack. We could remediate them, I think this might include needing to update `uuid` attribute + going in and update the data blob too https://github.com/sul-dlss/vatican_exhibits/blob/master/db/schema.rb#L16